src/main/java/ru/practicum/shareit/user/dao/InMemoryUserStorage.java

```diff
@@ -0,0 +1,72 @@
+package ru.practicum.shareit.user.dao;
+
+import org.springframework.stereotype.Repository;
+import ru.practicum.shareit.user.model.User;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Repository
+public class InMemoryUserStorage implements UserStorage {
+
+    private HashMap<Integer, User> users = new HashMap<>();
```

Comment:
1) Может быть [final](https://vertex-academy.com/tutorials/ru/modifikator-final/)
2) 
```suggestion
    private Map<Integer, User> users = new HashMap<>();
```